boards/mulle/doc.txt

```diff
@@ -21,14 +21,14 @@ Use `BOARD=mulle` for building RIOT for this platform.
 https://github.com/eistec/mulle/wiki/Datasheets contains a list of relevant
 documentation for the components.
 
-| MCU    | MK60DN512VLL10 – Cortex-M4 |
-|-------|-----------------------------------|
-| RAM    | 64kB |
-| Flash  | 512kB|
-| radio chipset   | AT86RF212B, sub-GHz IEEE802.15.4 transceiver, similar to the AT86RF233 |
-| external flash memory  | Micron M25P16 16 Mbits external NOR flash, used for storing configuration, measurements and other slow changing non-volatile data |
-| external FRAM memory   | Cypress/Ramtron FM25L04B 4 Kbits external F-RAM, used for storing counters and other rapidly changing non-volatile data |
-| accelerometer  | ST micro LIS3DH MEMS accelerometer,  |
+| MCU                   | MK60DN512VLL10 – Cortex-M4                                                                                                        |
+|:--------------------- |:--------------------------------------------------------------------------------------------------------------------------------- |
+| RAM                   | 64KiB                                                                                                                             |
+| Flash                 | 512KiB                                                                                                                            |
+| radio chipset         | AT86RF212B, sub-GHz IEEE802.15.4 transceiver, similar to the AT86RF233                                                            |
+| external flash memory | Micron M25P16 16 Mbits external NOR flash, used for storing configuration, measurements and other slow changing non-volatile data |
+| external FRAM memory  | Cypress/Ramtron FM25L04B 4 Kbits external F-RAM, used for storing counters and other rapidly changing non-volatile data           |
```

Comment:
Should this '4 Kbits' also be updated? From the datasheet the bits are arranged in 512 by 8. So then it would be `4 Kib`. With emphesis on the lowercase b. Or just `512 KiB`?

Reply:
I just converted it to byte (and the 16Mib also). I guess having the distinguish between b and B is tricky, so just using byte everywhere might be easier.